crates/sui-bridge-cli/src/lib.rs

```diff
@@ -494,6 +495,15 @@ impl LoadedBridgeCliConfig {
 #[derive(Parser)]
 #[clap(rename_all = "kebab-case")]
 pub enum BridgeClientCommands {
+    #[clap(name = "deposit-native-ether-on-eth")]
```

Comment:
@Bridgerz can you please take a look at this?

Reply:
it's an alternative of write to contract in etherscan but also works for local node run by anvil